README.md

```diff
@@ -1,4 +1,4 @@
-![OpenSearch logo](OpenSearch.svg)
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
```

Comment:
@KrooshalUX This should also have an `alt` attribute right?

Reply:
> @KrooshalUX This should also have an `alt` attribute right?
It's a markdown link now with the `alt` attribute

Reply:
Please use "OpenSearch Dashboards" as the alt content. Thanks!